oci-unit-tests/nginx_test.sh

```diff
@@ -0,0 +1,118 @@
+# shellcheck shell=dash
+
+. "$(dirname "$0")/helper/test_helper.sh"
+
+# cheat sheet:
+#  assertTrue $?
+#  assertEquals 1 2
+#  oneTimeSetUp()
+#  oneTimeTearDown()
+#  setUp() - run before each test
+#  tearDown() - run after each test
+
+# The name of the temporary docker network we will create for the
+# tests.
+readonly DOCKER_NETWORK=nginx_test
```

Comment:
AFAICS we do not need this docker network, we are not using more than one container in these test cases. We should remove this variable, the creation and destruction of the network, and the `--network` parameter from the `docker run` call.

Reply:
I added a reverse proxy test that makes use of the Docker network :)